source/clear-linux/getting-started/getting-started.rst

```diff
@@ -1,13 +1,13 @@
 Getting started
 ###############
 
+There are a number of options available for downloading and running Clear
+Linux\* OS for Intel® Architecture. You can install |CL| under Virtual
+Machines or containers, on Bare metal, or by way of Live image.
```

Comment:
@miguelinux The content of these deleted files exist elsewhere in the repo. For example, in the `Bare metal installation tutorial` there is more accurate and clear instructions for creating a bootable USB. Once the structural change is accepted I will be moving this content to `Getting Started`.